### PR TITLE
neocna/distributed-flooding

### PR DESCRIPTION
**Pull request by malkhamis-panw**
Cleanups and new flags to trigger distributed flooding

### CODE REVIEW

File: specs/+floodnodeuid.spec [hunk 1]
```diff
@@ -12,14 +12,13 @@ model:
 # Attributes
 attributes:
   v1:
-  - name: networkAddress
-    description: The network address of the node described by the NodeUID.
+  - name: nID
```
Comment (t00f) [inline specs/+floodnodeuid.spec:15]: I would prefer calling it nodeID then. I saw NID in the code and it does not mean anything

Comment (malkhamis-panw) [reply specs/+floodnodeuid.spec:15]: I would love to use ID... but that might conflict with Mongo if we ever decide to store.... but NodeID will always be like `nodeUID.NodeID` too wordy.... I don't know... let me do it tomorrow then

**Review by t00f — APPROVED** [review]

**Comment by malkhamis-panw** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "dist-flooding-v01",
    "component": "gaia",
    "pr-id": "898",
    "commit-sha": "ca71d4484ba0a4f857603834a338137f8e1d6d95",
    "pipeline": "master",
    "org": "aporeto-inc"
  },
  {
    "project": "dist-flooding-v01",
    "component": "pccnabackend",
    "pr-id": "210",
    "commit-sha": "59dd2003f5ad9c89810954f9be562503a666cbd3",
    "pipeline": "master",
    "org": "prismacloud/pcn"
  }
]
```

**Comment by malkhamis-panw** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "dist-flooding-v01",
    "component": "gaia",
    "pr-id": "898",
    "commit-sha": "ca71d4484ba0a4f857603834a338137f8e1d6d95",
    "pipeline": "master",
    "org": "aporeto-inc"
  },
  {
    "project": "dist-flooding-v01",
    "component": "pccnabackend",
    "pr-id": "210",
    "commit-sha": "1681f3afc958c280b5b1c5ec75ba7d54d8b75a89",
    "pipeline": "master",
    "org": "prismacloud/pcn"
  }
]
```

**Review by t00f — APPROVED** [review]
